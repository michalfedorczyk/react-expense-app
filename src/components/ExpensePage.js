import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpensesTotal from "./ExpensesTotal";

const ExpensePage = () => (
  <div>
    <ExpensesTotal />
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default ExpensePage;
