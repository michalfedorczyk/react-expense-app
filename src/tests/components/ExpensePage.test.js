import React from "react";
import { shallow } from "enzyme";
import ExpensePage from "../../components/ExpensePage";

test("should render ExpensePage correctly", () => {
  const wrapper = shallow(<ExpensePage />);
  expect(wrapper).toMatchSnapshot();
});
