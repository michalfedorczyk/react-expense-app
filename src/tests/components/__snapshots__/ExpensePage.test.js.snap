// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensePage correctly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
